### PR TITLE
fix: read version from package.json instead of changelog.md

### LINKED ISSUE #297: read version from package.json instead of changelog.md
ci config: 
https://github.com/twreporter/plate/blob/3b2fb97a5bfc55d94671855c14deffd8faeffd5d/.circleci/config.yml#L86

solution:
https://gist.github.com/DarrenN/8c6a5b969481725a4413

### PR DESCRIPTION
**Pull request by duidae**
fix #297 & jira [#319](https://twreporter-org.atlassian.net/browse/TWREPORTER-319)
may need some test, can review later.

ref: https://gist.github.com/DarrenN/8c6a5b969481725a4413

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -83,7 +83,7 @@ jobs:
 
       - run:
           name: Save pkg version to a file
-          command: node ./.circleci/scripts/read-changelog.js > .pkg-version
+          command: node -pe "require('./package.json').version" > .pkg-version
```
Comment (Aylie-Chou) [inline .circleci/config.yml:86]: In [neticrm repo](https://github.com/twreporter/neticrm/blob/d588bfe1ce1410a813f397cc5a63b77e06f25bc1/.circleci/config.yml#L12), I use `npm show . version` to get version, I think we could discuss a single way and apply it all. 
How do u think about it ?

Comment (duidae) [reply .circleci/config.yml:86]: `npm show . version` is also a good way but the package needs to be published to npm registry, for example, `plate` repo is not published and `npm show . version` gets error.

Comment (duidae) [reply .circleci/config.yml:86]: btw, I tried `npm show . version` in netiCRM repo & the version is actually from this [library](https://www.npmjs.com/package/sync-tool), which is irrelevant to our repo

**Review by Aylie-Chou — APPROVED** [review]
LGTM 👍

**Pull request merged**